packages/primitives/src/icons/discord-16.svg

```diff
@@ -0,0 +1,21 @@
+<!--
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
```

Comment:
Not sure we can add license for third party logos that other business entities have copyrights for :)

Reply:
cc: @savindi7 @brionmario